### PR TITLE
CB-3174 Redbeams: talkative name for cf stack

### PR DESCRIPTION
**Pull request by gergopapi2**

### CODE REVIEW

File: redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/databaseserver/requests/AllocateDatabaseServerV4Request.java [hunk 1]
```diff
@@ -24,7 +24,9 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class AllocateDatabaseServerV4Request extends ProviderParametersBase {
 
-    @Size(max = 40, min = 5, message = "The length of the name must be between 5 to 40 inclusive")
+    public static final int RDS_NAME_MAX_LENGTH = 40;
+
+    @Size(max = RDS_NAME_MAX_LENGTH, min = 5, message = "The length of the name must be between 5 to 40 inclusive")
```
Comment (bergerdenes) [inline redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/databaseserver/requests/AllocateDatabaseServerV4Request.java]: min?

Comment (gergopapi2) [reply redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/databaseserver/requests/AllocateDatabaseServerV4Request.java]: Sorry, could you please describe?

Comment (bergerdenes) [reply redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/databaseserver/requests/AllocateDatabaseServerV4Request.java]: constant for max length constraint was extracted, as opposed to min lenght constraint

Comment (gergopapi2) [reply redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/databaseserver/requests/AllocateDatabaseServerV4Request.java]: Thanks. Only the max length was reused in the code elsewhere.

Comment (jadair-cloudera) [reply redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/databaseserver/requests/AllocateDatabaseServerV4Request.java]: The message still has 40 hard coded.

**Pull request merged**